*** auto-tests/A-04-04-M-ok.out	2020-10-28 21:42:18.712439200 +0000
--- auto-tests/A-04-04-M-ok.outhyp	2020-11-25 10:39:14.755793000 +0000
***************
*** 17,28 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: e4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0
! ER|Abdul Figueiredo|Casablanca, Morocco|NORMAL|0|0
! h2o|Rohit Figueiredo|New Delhi, India|NORMAL|0|0
  H2SO4|John Figueiredo|Wellington, New Zealand|NORMAL|0|0
- o9|Hellen Figueiredo|San Francisco, CA, USA|NORMAL|0|0
  R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 17,28 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: ER|Abdul Figueiredo|Casablanca, Morocco|NORMAL|0|0
  H2SO4|John Figueiredo|Wellington, New Zealand|NORMAL|0|0
  R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
+ e4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0
+ h2o|Rohit Figueiredo|New Delhi, India|NORMAL|0|0
+ o9|Hellen Figueiredo|San Francisco, CA, USA|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
